repository/mock_repo.go

```diff
@@ -96,15 +96,22 @@ func (r *mockRepoForTest) ReadConfigString(key string) (string, error) {
 	return val, nil
 }
 
-func (r *mockRepoForTest) RmConfigs(keyPrefix string) error {
+// RmConfig remove all key/value pair matching the key prefix
+func (r *mockRepoForTest) RmConfig(k string) error {
 	for key := range r.config {
```

Comment:
Would it be better to mock `mockRepoForTest.config` as `map[string]map[string]string`. That's the only way i see to create mocks for `RmConfigSection`

Reply:
yes !